config/locales/en.yml

```diff
@@ -19,11 +19,11 @@ en:
 
     form:
       feedback: https://docs.google.com/forms/d/e/1FAIpQLSePiIC7xO1Dt6oxvIdL6jcGrlUkRDJDS1kxOcqvK-uspBED9w/viewform
```

Comment:
This link will need replacing too, it's the one that's shown at the top of each page under the navigation bar.